css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+
+  /* your code here */
+
+  height: 500px;
+  position: fixed;
+  left: 487px;
+}
```

Comment:
You shouldn't need a height property here. But to get the text centered I want you to use positioning properties as well.